tests/load-tests/errors.py

```diff
@@ -26,6 +27,7 @@
     "Couldnt get task via buldles resolver from quay.io due to 429": r"Message:.*Couldn't retrieve Task .*resolver type bundles.*https://quay.io/.* status code 429 Too Many Requests",
     "Couldnt get task via git resolver from gitlab.cee due to 429": r"Message:.*Couldn't retrieve Task .*resolver type git.*https://gitlab.cee.redhat.com/.* status code: 429",
     "Couldnt get task via http resolver from gitlab.cee": r"Message:.*Couldn't retrieve Task .*resolver type http.*error getting.*requested URL .*https://gitlab.cee.redhat.com/.* is not found",
+    "Error deleting on-pull-request default PipelineRun": r"Repo-templating workflow component cleanup failed: Error deleting on-pull-request default PipelineRun in namespace .*: Unable to list PipelineRuns for component .* in namespace .*: context deadline exceeded",
```

Comment:
Error reason here should be more `Timeout listing pipeline runs`.

Also regarding bug report for this: do we have some more details why it failed? Were there some `Listing pipeline runs failed, trying again, error was: ...` messages? Could you give an example of a job where it failed?

Reply:
Could you give an example of a job where it failed?